mobile-app/app/screens/AppNavigator/screens/Dex/DexAddLiquidityV2.tsx

```diff
@@ -0,0 +1,657 @@
+        </View>
+
+        <View style={tailwind('mt-5 mx-7')}>
+          <ButtonV2
+            fillType='fill'
+            label={translate('components/Button', 'Continue')}
+            styleProps='w-full'
+            disabled={!canContinue}
+            onPress={onSubmit}
+            testID='button_continue_add_liq'
+          />
+        </View>
+
+        {Platform.OS === 'web'
+          ? (
+            <BottomSheetWebWithNavV2
+              modalRef={containerRef}
+              screenList={ViewPoolContents}
+              isModalDisplayed={isModalDisplayed}
+              // eslint-disable-next-line react-native/no-inline-styles
+              modalStyle={{
+                position: 'absolute',
+                bottom: '0',
+                height: '404px',
+                width: '375px',
+                zIndex: 50,
+                borderTopLeftRadius: 15,
+                borderTopRightRadius: 15,
+                overflow: 'hidden'
+              }}
+            />
+          )
+          : (
+            <BottomSheetWithNavV2
+              modalRef={bottomSheetRef}
+              screenList={ViewPoolContents}
+              snapPoints={modalSortingSnapPoints}
+            />
+          )}
+      </ThemedScrollViewV2>
+    </View>
+  )
+}
+
+function AddLiquidityInputCard (
+  props: {
+    balance: BigNumber
+    type: 'primary' | 'secondary'
+    symbol: string
+    onPercentageChange: (amount: string, type: AmountButtonTypes) => void
+    onChange: (amount: string) => void
+    current: string
+    status?: TransactionCardStatus
+    setIsInputFocus: any // TODO: type checking
+    showInsufficientTokenMsg: boolean
+    showUTXOFeesMsg: boolean
+    hasInputAmount?: boolean
+  }): JSX.Element {
+  const Icon = getNativeIcon(props.symbol)
+  const isFocus = props.setIsInputFocus
+  return (
+    <>
+      <TransactionCard
+        maxValue={props.balance}
+        onChange={(amount, type) => {
+          props.onChange(amount)
+          props.onPercentageChange(amount, type)
+        }}
+        status={props.status}
+        amountButtonsStyle={tailwind('border-t-0.5')}
+        containerStyle={tailwind('pl-5 pr-5 pt-2 mr-px rounded-t-lg-v2')}
+      >
+        <ThemedViewV2
+          light={tailwind('border-mono-light-v2-300')}
+          dark={tailwind('border-mono-dark-v2-300')}
+          style={tailwind('flex flex-row items-center py-2')}
+        >
+          <View>
+            <Icon height={20} width={20} />
+          </View>
+          <WalletTransactionCardTextInput
+            onFocus={isFocus}
+            onBlur={isFocus}
+            onChangeText={txt => props.onChange(txt)}
+            placeholder='0.00'
+            value={props.current}
+            inputType='numeric'
+            displayClearButton={props.current !== ''}
+            onClearButtonPress={() => props.onChange('')}
+            titleTestID={`token_input_${props.type}_title`}
+            testID={`token_input_${props.type}`}
```

Comment:
```suggestion
            testID={`token_input_${props.type}`}
            inputContainerStyle={tailwind('pl-2')}
```
override padding left in <WalletTransactionCardTextInput />

Reply:
applied to add only since remove liqd need to account for the dual token.